message_ix/model/MESSAGE-MACRO_run.gms

```diff
@@ -114,6 +114,7 @@ Parameters
     price_diff_rel(*,node,sector,year_all)
 
     report_iteration(iteration,*)
+    trade_cost_detail(node, commodity, year_all)              'net of commodity import costs and commodity export revenues by commodity, node and year'
```

Comment:
Perhaps better to describe with the same order of indexes, i.e., "node, commodity, and year"